publisher-html/src/main/resources/templates/events/events.hbs

```diff
@@ -80,6 +80,17 @@
         gap: 4px;
     }
 
+    .accessibility-details {
+        display: flex;
+        background-color: white;
+        color: inherit;
+        position: absolute;
+        left: 8px;
```

Comment:
[nit] This project uses CSS variables to organize FE values as consistent as possible. Please have a look at `theme.hbs`, it includes a `--space-medium: 8px` variable that should be re-used for these props